### ISSUE TITLE
Concurrent Account Refreshing

### ISSUE THREAD

**Issue by crispinWeissfuss** [Rukenshia/saml2aws-auto#26]
Hey Rukenshia,

* Problem:
  * I have a huge list of accounts in a single group

* Feature Request
  * add a feature which refreshes accounts in parallel

* Alternative Solution
  * On user site I could split it up in several groups - but I like to group them logically

Thanks
